ansible/roles/amazon-cloudwatch-agent/tasks/redhat.yml

```diff
@@ -60,7 +65,7 @@
       ansible.builtin.template:
         src: oracle_health.sh.j2
         dest: /opt/collectd/oracle_health.sh
-        mode: 0500
+        mode: 0755
         owner: oracle
 
     - name: Make Oracle Health Check script executable
```

Comment:
I guess this step is redundant now as the above step copies with 0755 permissions

Reply:
Yep, that'll be me not having comprehended permissions quite yet +1